_configuration/configuration.md

```diff
@@ -606,7 +606,10 @@ Setting these too high will cause the corresponding stepper motor to lose steps,
 ```
 When the velocity of any axis changes, its acceleration (or deceleration) in mm/s/s is limited by the current max acceleration setting. Also see the *jerk* settings below, which specify the largest instant speed change that can occur between segments.
 
-A value of 3000 means that an axis may accelerate from 0 to 3000mm/m (50mm/s) within a one second movement.
+A value of e.g 50 means that an axis can accelerate from 0 to 50 mm/s in one second. 
```

Comment:
And maybe it should also say:
```
...means that an axis is allowed to accelerate or decelerate by up to 50mm/s (3000mm/m) within one second.
```
Since mm/m is the standard feed-rate unit that users see, we should refer to it in cases like this.

But note, the `F` parameter to GCode movement commands actually specifies feed-rate in current units, not necessarily mm/m. It could be inches/m.